acme/acme/challenges.py

```diff
@@ -216,6 +215,75 @@ def response_and_validation(self, account_key, *args, **kwargs):
 
 
 @ChallengeResponse.register
+class DNS01Response(KeyAuthorizationChallengeResponse):
+    """ACME "dns-01" challenge response."""
+    typ = "dns-01"
+
+    def simple_verify(self, chall, domain, account_public_key):
+        """Simple verify.
+
+        :param challenges.DNS01 chall: Corresponding challenge.
+        :param unicode domain: Domain name being verified.
+        :param account_public_key: Public key for the key pair
+            being authorized.
+        :param JWK account_public_key:
```

Comment:
I am not really a pythonist myself - but isn't this line excessive (no JWK arg on the list) ?